packages/next/compiled/webpack/minify.js

```diff
@@ -0,0 +1,273 @@
+"use strict";
+
+const {
+  minify: terserMinify
+} = require('terser');
```

Comment:
Looks like this asset was emitted without being ran through ncc. We probably need to upgrade to the new ncc with subbundle support?